### PR TITLE
Session ii 

### PR DESCRIPTION
**Pull request by mpricu**
Session_ii_session1_homework

### CODE REVIEW

File: session_ii/javascript-fundamentals/session1/homework/mpricu/module.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { array1 } from "./array.js";
```
Comment (ghost) [inline session_ii/javascript-fundamentals/session1/homework/mpricu/module.js]: Doesn't this give you an error? `array1` is exported as a default export.

File: session_ii/javascript-fundamentals/session1/homework/mpricu/area.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    constructor(sides, lenght) {
+        super(sides, lenght);
+        this.name = 'Square';
+    }
+    getArea() {
+        return this.lenght * this.lenght
+    };
+}
+const square = new Square(4, 10);
+console.log('Square area is', square.getArea());
```
Comment (ghost) [inline session_ii/javascript-fundamentals/session1/homework/mpricu/area.js:32]: The code looks good.

File: session_ii/javascript-fundamentals/session1/homework/mpricu/module.js (hunk 1 above)
Comment (mpricu) [reply session_ii/javascript-fundamentals/session1/homework/mpricu/module.js]: Yes, but in my initial file it was "export {array1}" :((

File: session_ii/javascript-fundamentals/session1/homework/mpricu/area.js (hunk 2 above)
Comment (mpricu) [reply session_ii/javascript-fundamentals/session1/homework/mpricu/area.js:32]: The triangle area calculation based on the perimeter and sides' length is named Heron's formula and I did not learn about it at the school :)

**Comment by mpricu** [comment]
session_ii_session1_homework_export_revision

**Pull request closed without merging**